app/portainer/components/template-list/template-list-controller.js

```diff
@@ -16,12 +16,26 @@ function TemplateListController($async, $state, DatatableService, Notifications,
     DatatableService.setDataTableTextFilters(this.tableKey, this.state.textFilter);
   };
 
+  this.filterByType = function (item) {
```

Comment:
that's a hell of a lot better - one small improvement would be to name it so the reader know what you mean by "type"

the remote orchestrator type, or the template type?